relays/bin-substrate/src/cli/bridge.rs

```diff
@@ -70,6 +75,29 @@ pub trait HeadersCliBridge: CliBridgeBase {
 	>;
 }
 
+/// Bridge representation that can be used from the CLI for relaying headers
+/// from a parachain to a relay chain.
+pub trait ParachainHeadersCliBridge: CliBridgeBase {
```

Comment:
We'll also have parachain <> parachain bridge soon (our final goal). So probably `RelayParachainHeadersCliBridge` instead? Could be postponed, though :)

Reply:
Done